python_src/src/lamp_py/performance_manager/gtfs_utils.py

```diff
@@ -28,16 +35,32 @@ def start_time_to_seconds(
     return int(hour) * 3600 + int(minute) * 60 + int(second)
 
 
+def start_timestamp_to_seconds(start_timestamp: int) -> int:
+    """
+    transform record with a timestamp and a service date into a seconds after midnight float
+    """
+    service_date_string = str(service_date_from_timestamp(start_timestamp))
+    year = int(service_date_string[:4])
+    month = int(service_date_string[4:6])
+    day = int(service_date_string[6:8])
+    start_of_service_day = BOSTON_TZ.localize(
+        datetime.datetime(year=year, month=month, day=day)
+    )
+
+    start_time = datetime.datetime.fromtimestamp(start_timestamp, tz=BOSTON_TZ)
+
+    time_after_midnight = start_time - start_of_service_day
+    return int(time_after_midnight.total_seconds())
```

Comment:
nitpick: I think we could avoid some timezone tom foolery here, but I might be wrong. Since `start_timestamp` is a UTC timestamp, once we have the `service_date_string` we could get a UTC timestamp for that day at midnight and then our return value should just be the difference of the two UTC timestamps. 

`
return int(start_timestamp - service_date_utc_timestamp)
`